person.rb

```diff
@@ -2,31 +2,37 @@
 
 class Person < Nameable
   attr_accessor :name, :age
```

Comment:
- Hey, :wave: , In here your setter/getter method should include the books and rentals. the attr_reader should just have the only the id as each time the person rents a book, each book is unique as a result of the id. `attr_accessor :name, :age, :books, :rentals`. please, update this.